app/assets/javascripts/admin/enterprise_groups/controllers/side_menu_controller.js.coffee

```diff
@@ -5,11 +5,11 @@ angular.module("admin.enterprise_groups")
 
     $scope.menu.setItems [
       { name: 'Primary Details', icon_class: "icon-user" }
-      { name: 'Users', icon_class: "icon-user" }
-      { name: 'About', icon_class: "icon-pencil" }
-      { name: 'Images', icon_class: "icon-picture" }
-      { name: 'Contact', icon_class: "icon-phone" }
-      { name: 'Web', icon_class: "icon-globe" }
+      { name: (t('users')), icon_class: "icon-user" }
```

Comment:
We need to keep that , otherwise we have an Error: ExecJS::RuntimeError: SyntaxError: [stdin]:14:36: unmatched )